modP2pImpl/build.xml

```diff
@@ -13,6 +13,7 @@
 		<pathelement location="${dir.lib}/commons-collections4-4.0.jar" />
 		<pathelement location="${dir.mod}/modP2p.jar" />
 		<pathelement location="${dir.mod}/modP2pImpl.jar" />
+		<pathelement location="${dir.mod}/modAionBase.jar" />
```

Comment:
Perhaps `IMsgHeader` should go in `modP2P` to avoid adding another dependency to this module